rasa/core/test.py

```diff
@@ -481,13 +489,38 @@ def emulate_loop_rejection(partial_tracker: DialogueStateTracker) -> None:
     partial_tracker.update(ActionExecutionRejected(rejected_action_name))
 
 
+def _get_processor_entity_evaluation_result(
+    processor: "MessageProcessor",
+    tracker: DialogueStateTracker,
+    prediction: PolicyPrediction,
+):
+    previous_event = tracker.events[-1]
+    if isinstance(previous_event, UserUttered):
+        entity_predictions = [
```

Comment:
How about making this very explicit?
```suggestion
        entities_predicted_by_policies = [
```